core/embed/rust/src/micropython/testutil.rs

```diff
@@ -11,8 +11,8 @@ static mut MPY_INITIALIZED: bool = false;
 /// Initialize the MicroPython environment.
 ///
 /// This is very hacky, in no way safe, and should not be used in production.
-/// The stack is configured on a best-effort basis and depending on from where
-/// this is called, you might get errorneous "recursion exceeded" problems.
+/// The stack is configured to span all of memory, effectively disabling the
+/// stack guard. I have no idea what can happen.
```

Comment:
Curiously, it doesn't seem to affect scanning for GC roots, if I'm reading the code correctly. Doesn't really matter for unit for tests though.